.github/ISSUE_TEMPLATE/tech-debt.md

```diff
@@ -0,0 +1,15 @@
+---
+name: "\U00002699 Tech Debt"
```

Comment:
@jurre I wasn't sure which fun emoji you thought would be best... this gear icon looked good. If you like it, maybe add it to the label itself just like is on the bug label?

Reply:
I dig it 👍